Task task			# See Task.msg
geometry_msgs/Point location 	# Location in which to perform the sweep